datadog/resource_datadog_integration_aws.go

```diff
@@ -6,13 +6,15 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+
 	datadogV1 "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 )
 
 var integrationAwsMutex = sync.Mutex{}
 
-func accountAndRoleFromID(id string) (string, string, error) {
+func AccountAndRoleFromID(id string) (string, string, error) {
```

Comment:
shouldn't this be moved to utils to follow the pattern ?

Reply:
Good catch, I thought I moved this but I only moved the `AccountAndNamespaceFromID` function. 
Moving now